src/main/java/controller/PassengersController.java

```diff
@@ -0,0 +1,124 @@
+package controller;
+
+import db.DatabaseHandler;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Timestamp;
+
+public class PassengersController extends DatabaseHandler {
+        DatabaseHandler databaseHandler = new DatabaseHandler();
+
+    public void listAllPassengers() {
+        Statement statement = databaseHandler.createStatement();
+        try {
+            ResultSet resultSetPassengers = statement.executeQuery("SELECT * FROM passengers");
+            System.out.println("=========================================");
+            System.out.println("Passengers table");
+
+            while (resultSetPassengers.next()) {
+                int myPassengerId = resultSetPassengers.getInt("id");
+                String myPassengerFullName = resultSetPassengers.getString("fullName");
+                String myPassengerEmail = resultSetPassengers.getString("email");
+                String myPassengerPhoneNumber = resultSetPassengers.getString("phoneNumber");
+                int myPassengerAge = resultSetPassengers.getInt("age");
+                int myPassengerStartLocationId = resultSetPassengers.getInt("startLocationId");
+                int myPassengerStopLocationId = resultSetPassengers.getInt("stopLocationId");
+                int myPassengerBusID = resultSetPassengers.getInt("busId");
+                Timestamp timestamp = resultSetPassengers.getTimestamp("createdOn");
+
+                System.out.println(myPassengerId
+                        + " " + myPassengerFullName
+                        + " " + myPassengerEmail
+                        + " " + myPassengerPhoneNumber
+                        + " " + myPassengerAge
+                        + " " + myPassengerStartLocationId
+                        + " " + myPassengerStopLocationId
+                        + " " + myPassengerBusID
+                        + " " + timestamp);
```

Comment:
Maybe just to make the output more beautiful and readable add dash between the values. 
`myPassengerId + " - " + myPassengerFullName`

Reply:
done